### DISCUSSION TITLE
generateClientApiV2  - maxProjectionDepth ignored?

### DISCUSSION THREAD

**Discussion by ehchn1** [Netflix/dgs-framework#1626] [General]
Hi,
I'm the midst of moving to "generateClientApiV2". 
I noticed that "maxProjectionDepth" seems to be ignored.

I monitored by doing the below:
1. Changing the value, then check the number of files generated - it always generate same amount of files
2. Looking at the codegen output - it didn't printout "maxProjectionDepth" as the parameter, suggesting it's ignored? The others flag are printed though eg: skip-entities, generate-data-types, generate-boxed-types

Anyone got any feedbacks regarding this observation?

Thanks.

**Comment by srinivasankavitha** [comment]
Yes, that setting is not relevant for v2 API since the mechanism is different. You can ignore it altogether.
